midas-contracts/contracts/abstract/WithSanctionsList.sol

```diff
@@ -12,6 +12,11 @@ import "./MidasInitializable.sol";
  * @author RedDuck Software
  */
 abstract contract WithSanctionsList is WithMidasAccessControl {
+    /**
+     * @dev leaving a storage gap for futures updates
+     */
+    uint256[50] private __gap;
```

Comment:
Move this gap behind the definition of `address public sanctionsList;` behind it.

Reply:
https://github.com/RedDuck-Software/midas-contracts/pull/73

Reply:
LGTM